### ISSUE TITLE
Issue with market.yandex.ru

### ISSUE THREAD

**Issue by farag2** [uBlock-LLC/uBlock#614]
Ublock 0.8.6.0 beta 4. uBlock blocks all pics on http://market.yandex.ru

**Comment by gorhill** [comment]
EasyPrivacy blocks `awaps.yandex.ru`. You need to report to the maintainer of EasyPrivacy that valid contents is blocked: https://forums.lanik.us/viewforum.php?f=64.

I will add an narrow exception filter (just for images) because hpHosts also blocks `awaps.yandex.ru`.

**Edit:** actually I had to counter more than one block filter:

```
@@||awaps.yandex.ru^$domain=market.yandex.ru
@@||mc.yandex.ru^$domain=market.yandex.ru
@@||yastatic.net/market-export/*/advert.js$domain=market.yandex.ru
```

If you could report to EasyPrivacy, that would be nice to users of other blockers.

**Comment by gorhill** [comment]
Works on uBlock 0.8.6.0-beta.4 Chromium.

Does not work on uBlock 0.8.6.0-beta.4 Firefox due to [another issue](https://github.com/gorhill/uBlock/commit/263b756764829eebb877f1b5b92bba3da5cf28e0#commitcomment-9459495): will close when Firefox issue resolved.

**Comment by gorhill** [comment]
Firefox issue fixed with 263b756764829eebb877f1b5b92bba3da5cf28e0
